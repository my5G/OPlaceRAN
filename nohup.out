Starting to serve on 127.0.0.1:8001
Starting to serve on 127.0.0.1:8001
E0217 20:46:25.116907   10625 proxy_server.go:144] Error while proxying request: context canceled
E0217 20:46:37.401764   10625 proxy_server.go:144] Error while proxying request: context canceled
Starting to serve on 127.0.0.1:8001
Starting to serve on 127.0.0.1:8001
Starting to serve on 127.0.0.1:8001
