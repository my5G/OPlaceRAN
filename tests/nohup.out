/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
mkdir: cannot create directory ‘2’: File exists
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
/root/RANPlacer/k8s-operators/config/samples/ran_v1alpha1_ranplacer.yaml
ranplacer.ran.unisinos/ranplacer-sample created
ranplacer.ran.unisinos "ranplacer-sample" deleted
